src/GitHub.VisualStudio/Menus/CopyLink.cs

```diff
@@ -45,7 +48,8 @@ public void Activate([AllowNull]object data = null)
 
         public bool CanShow()
         {
-            return IsGitHubRepo();
+            return IsGitHubRepo().Wait(250); 
```

Comment:
Just looked up the return value `Wait()`, definitely not what I was looking for!